backend/fleet_management/admin.py

```diff
@@ -18,22 +18,27 @@ def dehydrate_passengers(self, drive):
             for passanger in drive.passengers.all()
         )
 
+    fuel_consumption = Field(attribute='fuel_consumption')
+
     class Meta:
         model = Drive
         fields = (
             "id",
-            "driver",
-            "car__plates",
             "passengers",
             "date",
             "start_mileage",
             "end_mileage",
             "description",
             "start_location",
             "end_location",
+            "driver",
+            "fuel_consumption",
+            "car__plates",
             "project__title",
         )
 
+        export_order = fields
```

Comment:
Isn't it used by default?

Reply:
Defining export_order is necessary to keep an earlier order. The change described here https://github.com/CodeForPoznan/pah-fm/pull/280#discussion_r308967346 makes fuel_consumption land as the first column which we don't want to.